regcore_read/views/es_search.py

```diff
@@ -37,6 +39,10 @@ def search(request, doc_type):
             term['version'] = version
         if regulation:
             term['regulation'] = regulation
+        if is_root:
+            term['is_root'] = is_root
+        if is_subpart:
+            term['is_subpart'] = is_subpart
```

Comment:
I think the API should generally complain about badly formatted data. The web UI should be more forgiving.